libs/ui-toolkit/src/components/nav/trading/trading-nav.tsx

```diff
@@ -0,0 +1,93 @@
+import classnames from 'classnames';
+import { VegaLogo } from '../../vega-logo';
+import type { MouseEvent } from 'react';
+
+interface NavItemProps {
```

Comment:
@mattrussell36 yeah not implemented yet, it's noted in the PR description. It looks like the guys might use it in the simple trading app, so thought it could go here for now. If they don't go with it then I'm happy to move it later on.